File: swagger/models/gift_card_layout.py
from .base_model import BaseModel


class GiftCardLayout(BaseModel):
	"""
	Gift card layout

	Attributes:

		layout_id
		The ID of the layout.

		layout_name
		The name of the layout.
	"""

	input_map = {
		'LayoutId': 'layout_id',
		'LayoutName': 'layout_name',
		}

	output_map = {
		'layout_id': 'LayoutId',
		'layout_name': 'LayoutName',
		}

	layout_id = None
	layout_name = None

File: swagger/Models/GiftCardLayout.php
<?php

namespace App\Mindbody\Models;

/**
 * Gift card layout
 */
class GiftCardLayout extends BaseModel
{
	/**
	 * The ID of the layout.
	 * 
	 * @var integer format:int32
	 */
	public $layoutId;
	/**
	 * The name of the layout.
	 * 
	 * @var string 
	 */
	public $layoutName;

	protected function getInputMap()
	{
		return [
			'LayoutId' => ['layoutId', 'integer', 'int32'],
			'LayoutName' => ['layoutName', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'layoutId' => ['LayoutId', 'integer', 'int32'],
			'layoutName' => ['LayoutName', 'string', null],
			];
	}
}
